exporter/collector/metrics.go

```diff
@@ -119,8 +119,14 @@ func NewGoogleCloudMetricsExporter(
 	version string,
 	timeout time.Duration,
 ) (*MetricsExporter, error) {
-	view.Register(MetricViews()...)
-	view.Register(ocgrpc.DefaultClientViews...)
+	err := view.Register(MetricViews()...)
```

Comment:
Does this mean you can't use more than one googlecloud exporter in a collector?

Reply:
does that cause `view.Register()` to throw an error? if so we can change this

Reply:
I believe that does.  You can test it by adding a second view.Register() call in this function :).

Reply:
Per offline discussion, tried adding a double call and didn't see the expected failure. So, reverting this and skipping errcheck on these lines (ie, current behavior) until we can understand why this didn't fail when we expected it to.